dynamiqs/utils/states.py

```diff
@@ -97,6 +97,28 @@ def fock_dm(
     return todm(fock(dim, number, dtype=dtype, device=device))
 
 
+def basis(
+    dim: int | tuple[int, ...],
+    number: int | tuple[int, ...],
+    *,
+    dtype: torch.complex64 | torch.complex128 | None = None,
+    device: str | torch.device | None = None,
+):
+    """Alias for [`dq.fock()`](/python_api/utils/states/fock.html)."""
```

Comment:
Correct syntax is
```suggestion
    """Alias for [`dq.fock()`][dynamiqs.fock]."""
```

Reply:
I tried this but the link was broken for some reason, both in the main function page or in the API summary table.

Reply:
Nevermind it works if you use `[dynamiqs.fock]` and I was using `(dynamiqs.fock)`. Fixed.